DataLineage/README.md

```diff
@@ -0,0 +1,39 @@
+# Google Cloud Data Catalog Lineage for PHP
+
+> Idiomatic PHP client for [Google Cloud Data Catalog Lineage](https://cloud.google.com/data-catalog/docs/concepts/about-data-lineage).
+
+[![Latest Stable Version](https://poser.pugx.org/google/cloud-datalineage/v/stable)](https://packagist.org/packages/google/cloud-datalineage) [![Packagist](https://img.shields.io/packagist/dm/google/cloud-datalineage.svg)](https://packagist.org/packages/google/cloud-datalineage)
+
+* [API documentation](https://cloud.google.com/php/docs/reference/cloud-datalineage/latest)
+
+**NOTE:** This repository is part of [Google Cloud PHP](https://github.com/googleapis/google-cloud-php). Any
+support requests, bug reports, or development contributions should be directed to
+that project.
+
+### Installation
+
+To begin, install the preferred dependency manager for PHP, [Composer](https://getcomposer.org/).
+
+Now to install just this component:
+
+```sh
+$ composer require google/cloud-datalineage
+```
+
+### Authentication
+
+Please see our [Authentication guide](https://github.com/googleapis/google-cloud-php/blob/main/AUTHENTICATION.md) for more information
+on authenticating your client. Once authenticated, you'll be ready to start making requests.
+
+### Sample
+
+```php
```

Comment:
Just want to make sure: Should this be empty?

Reply:
thank you for pointing this out! It's now fixed.